package.json

```diff
@@ -51,6 +51,7 @@
     "@oclif/command": "^1.0.0",
     "@oclif/config": "^1.0.0",
     "@oclif/plugin-help": "^2.0.0",
+    "@thiagoveras/plugin-update": "^1.0.0",
```

Comment:
In the future we can fork this project under vtex name 🚀 
(I'm in the wrong here, because of `@tiagonapoli/oclif-plugin-spaced-commands` - it was supposed to be a temporary solution because of toolbelt 3.0 😢 )

Reply:
This is just a testing plugin, the final result will be under vtex name